data/sles4sap/qe_sap_deployment/mr_test_ec2.yaml

```diff
@@ -0,0 +1,33 @@
+provider: "ec2"
+apiver: 2
```

Comment:
As it is a new file, why not to give a try to the apiver:3 and **hana_media** ?  (Just a suggestion, no need to change)

Reply:
> **hana_media**
I referred `data/sles4sap/qe_sap_deployment/qesap_aws.yaml` so I used `apiver: 2`.
Thanks for the reminder. Good to know.